index.dev.js

```diff
@@ -76,6 +76,16 @@ async function startInsect() {
   if (interactive) {
     var [fs, clipboardy, readline, xdgBasedir] = await Promise.all([import("fs"), import("clipboardy"), import("readline"), import("xdg-basedir")]);
 
+    // Create `xdgBasedir.data` if it doesn't already exist.
+    // See https://github.com/sharkdp/insect/issues/364.
+    try {
+      fs.mkdirSync(xdgBasedir.data, { recursive: true });
```

Comment:
Should we maybe change the permissions of `xdgBasedir.data` on creation to make it non-world-accessable? It could be argued that that's not Insect's responsibility though.